cmd/session-v8.go

```diff
@@ -375,6 +375,18 @@ func (s sessionV8) CloseAndDie() {
 	console.Fatalln("Session safely terminated. To resume session `mc session resume " + s.SessionID + "`")
 }
 
+func (s sessionV8) copyCloseAndDie(sessionFlag bool) {
+	if sessionFlag {
+		s.Close()
+		console.Fatalln("Command terminated safely. Run this command to resume copy again.")
+	} else {
+		s.mutex.Lock()
```

Comment:
question: why do we need to do anything at all when session flag wasn't passed along with `mc cp ...`?

Reply:
Not sure. I am just keeping session behaviour same as `CloseAndDie()`. When `mc session` command is removed, type `sessionV8` may clean up.